pkg/cfgstruct/bind.go

```diff
@@ -53,6 +53,10 @@ func bindConfig(flags FlagSet, prefix string, val reflect.Value,
 
 	for i := 0; i < typ.NumField(); i++ {
 		field := typ.Field(i)
+		if field.Tag == "-:\"\"" {
```

Comment:
Was there some problem with just using `-`? Maybe we can use a keyword `ignore`? PS: you can use backquote to avoid escaping ". https://golang.org/ref/spec#String_literals

Specifying:
```
   Field `-:""`
```

Would look very weird.